### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by gszabo** [Python-Cardano/pycardano#397] [General]
What is the release schedule in this library?

An issue reported by us has been fixed by https://github.com/Python-Cardano/pycardano/pull/380, but a new version hasn't been released, meaning we still have to include a workaround in the code.

What can we do the make a new release happen? Thank you in advance for your help.

**Comment by cffls** [comment]
Hi @gszabo , sorry about the late response. The patch `v0.12.1` has been release: https://pypi.org/project/pycardano/. This project doesn't have a regular release schedule in the past, but I think it makes sense to establish one, e.g. monthly release for bug fixes and improvements.

**Reply by gszabo** [reply]
Thank you for the release!
